hack/lib/version.sh

```diff
@@ -135,40 +144,38 @@ kube::version::load_version_vars() {
   source "${version_file}"
 }
 
-kube::version::ldflag() {
-  local key=${1}
-  local val=${2}
-
-  # If you update these, also update the list pkg/version/def.bzl.
-  echo "-X '${KUBE_GO_PACKAGE}/pkg/version.${key}=${val}'"
-  echo "-X '${KUBE_GO_PACKAGE}/vendor/k8s.io/client-go/pkg/version.${key}=${val}'"
-  echo "-X '${KUBE_GO_PACKAGE}/pkg/kubectl/version.${key}=${val}'"
-}
-
 # Prints the value that needs to be passed to the -ldflags parameter of go build
 # in order to set the Kubernetes based on the git tree status.
 # IMPORTANT: if you update any of these, also update the lists in
 # pkg/version/def.bzl and hack/print-workspace-status.sh.
 kube::version::ldflags() {
   kube::version::get_version_vars
 
-  local buildDate=
-  [[ -z ${SOURCE_DATE_EPOCH-} ]] || buildDate="--date=@${SOURCE_DATE_EPOCH}"
```

Comment:
all of this logic amounts to either expanding to a `--date=@${SOURCE_DATE_EPOCH}` flag if `$SOURCE_DATE_EPOCH` is set, or otherwise expanding to nothing (as an arg or no arg to `date ...`).

we can just inline this and use parameter substitution to accomplish that explicitly _and_ actually quote it